### ISSUE TITLE
Deprecation warning in Ruby 3.0.3

### ISSUE THREAD

**Issue by kaiserbrito** [tilo/smarter_csv#184]
Hey,

Using Ruby 3.0.3 and smarter_csv 1.4.0 a deprecation warning is raised:
```
DEPRECATION WARNING: /ruby/3.0.3/lib/ruby/gems/3.0.0/gems/smarter_csv-1.4.0/lib/smarter_csv/smarter_csv.rb:260: warning: `$INPUT_RECORD_SEPARATOR' is deprecated
```

Thanks a lot!

**Comment by tilo** [comment]
awesome - and they don't mention what to do instead :P

https://blog.saeloun.com/2020/03/09/ruby-2-7-setting-special-variables-to-non-nil-value-is-warned.html

**Comment by tilo** [comment]
thank you @jfouse !
@kaiserbrito this will be fixed in 1.5.0
